packages/js/manifests/polywrap/src/formats/polywrap.app/migrators/0.2.0_to_0.3.0.ts

```diff
@@ -0,0 +1,15 @@
+import { ILogger } from "@polywrap/logging-js";
+import { AppManifest as OldManifest } from "../0.2.0";
+import { AppManifest as NewManifest } from "../0.3.0";
+
+export function migrate(migrate: OldManifest, logger?: ILogger): NewManifest {
+  if (migrate.extensions?.meta) {
+    logger?.warn(
+      `The Polywarp Meta Manifests has been deprecated for app projects.`
```

Comment:
The Polywarp Meta **Manifest** has been deprecated for plugin projects.

Grammar :grin: